src/shared/components/case-viewer/case-viewer.component.spec.ts

```diff
@@ -1214,3 +1250,119 @@ describe('CaseViewerComponent - prependedTabs', () => {
     expect((<HTMLElement>tasksTab.querySelector('.mat-tab-label-content')).innerText).toBe('Tasks');
   });
 })
+
+// noinspection DuplicatedCode
+describe('CaseViewerComponent - Overview with prepended Tabs', () => {
+  let mockLocation: any;
+
+  let comp: CaseViewerComponent;
+  let f: ComponentFixture<CaseViewerComponent>;
+  let d: DebugElement;
```

Comment:
Minor change - Could we change f to component and d to element or similar please?

Reply:
good shout I have copied paste it, now it's changing to something meaningful.